test/osde2e/suite_test.go

```diff
@@ -90,9 +90,15 @@ var _ = AfterSuite(func() {
 	Expect(err).ToNot(HaveOccurred())
 })
 
-func setVars(possibleWN, possibleNS string) {
-	os.Setenv("WATCH_NAMESPACE", possibleWN)
-	os.Setenv("SKIP_FLAKES", "true")
+func setVars(possibleWN, possibleNS string, t *testing.T) {
+	err := os.Setenv("WATCH_NAMESPACE", possibleWN)
+	if err != nil {
+		t.Fatalf("Failed to set WATCH_NAMESPACE env var %s", err)
+	}
+	err = os.Setenv("SKIP_FLAKES", "true")
+	if err != nil {
+		t.Fatalf("Failed to set SKIP_FLAKES env var %s", err)
```

Comment:
I don't think not setting the `SKIP_FLAKES` should be a fatal error. This would case the test that should be skipped to run but the operator can still be installed. My view would be to log that the `SKIP_FLAKES` did not get set but continue to run the test. You never know the flaky test might pass in the end.

Reply:
Done, good idea!